Makefile

```diff
@@ -88,7 +88,7 @@ endif
 ifndef HAS_MINIKUBE
 	$(error "Please install Minikube v1.0.0 or higher")
 endif
-	minikube status --profile ${MINIKUBE_PROFILE} || minikube start --profile ${MINIKUBE_PROFILE} --vm-driver ${MINIKUBE_DRIVER} --cpus ${MINIKUBE_CPUS} --memory ${MINIKUBE_MEMORY} --disk-size ${MINIKUBE_DISKSIZE} --feature-gates="TTLAfterFinished=true"
+	minikube status --profile ${MINIKUBE_PROFILE} || minikube start --kubernetes-version='v1.15.4' --profile ${MINIKUBE_PROFILE} --vm-driver ${MINIKUBE_DRIVER} --cpus ${MINIKUBE_CPUS} --memory ${MINIKUBE_MEMORY} --disk-size ${MINIKUBE_DISKSIZE} --feature-gates="TTLAfterFinished=true"
```

Comment:
Perhaps make it a configuration variable, so that people on older minikube (e.g. 1.0 should still work!) can run the script still easily.  E.g. `MINIKUBE_KUBERNETES` like the other `MINIKUBE_DRIVER` and friends.

Reply:
Updated